common/mails/courses/index.ts

```diff
@@ -230,4 +235,15 @@ export async function sendPupilCourseSuggestion(course: Course | Prisma.course,
             courseURL: `https://app.lern-fair.de/single-course/${subcourse.id}`,
         });
     }
+    for (let pupil of pupils) {
+        if (!courseSubject) {
+            notify(pupil);
```

Comment:
should everybody be matched to a course with no subject? or better to not notify?

Reply:
Yes, courses without subject are meant for all pupils